sklearn/preprocessing/data.py

```diff
@@ -1,10 +1,12 @@
+# coding: utf-8
```

Comment:
unrelated to this PR. Just looked at other files in sklearn and it's sometimes there and sometimes not. It seems there's no strong convention about that.